### ISSUE TITLE
Stress tests taking forever

### ISSUE THREAD

**Issue by epicfaace** [codalab/codalab-worksheets#4395]
See https://github.com/codalab/codalab-worksheets/actions/runs/4089166286/jobs/7051586866.

Maybe change parameter for worker timeout?

**Comment by AndrewJGaut** [comment]
TODO: Add updates

Could speed up by doing some things in parallel (e.g. cpu, gpu, etc.) @epicfaace will make an issue

**Comment by epicfaace** [comment]
Closing in favor of https://github.com/codalab/codalab-worksheets/issues/4402
